tests/integration/error_warning_file_tests.py

```diff
@@ -0,0 +1,521 @@
+import os
+import csv
+
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.config import CONFIG_SERVICES
+from dataactcore.models.jobModels import Submission
+from dataactcore.models.domainModels import concat_tas_dict
+from tests.unit.dataactcore.factories.domain import SF133Factory, TASFactory
+from dataactvalidator.health_check import create_app
+from dataactvalidator.validation_handlers.validationManager import ValidationManager
+from dataactcore.models.lookups import (FILE_TYPE_DICT, JOB_TYPE_DICT, JOB_STATUS_DICT)
+from dataactbroker.handlers.fileHandler import report_file_name
+
+from tests.integration.baseTestValidator import BaseTestValidator
+from tests.integration.integration_test_helper import insert_submission, insert_job
+
+FILES_DIR = os.path.join('tests', 'integration', 'data')
+
+# Valid Files
+APPROP_FILE_T = os.path.join(FILES_DIR, 'appropValid.csv')
```

Comment:
What does the `T` stand for?

Reply:
Copied from `fileTests`. I believe stands for "Test"? Can be renamed to remove the T.